features/include/pcl/features/rift.h

```diff
@@ -80,10 +80,10 @@ namespace pcl
 
 
       /** \brief Empty constructor. */
-      RIFTEstimation () : gradient_ ()
+      RIFTEstimation () : gradient_ (), nr_distance_bins_ (4), nr_gradient_bins_ (8)
```

Comment:
Why are they added here? They are already default member initialized?

Reply:
Most of these are merge issues from the last rebase with master; fixed.